pkg/formatted/k8s.go

```diff
@@ -29,11 +31,25 @@ var ConditionColor = map[string]color.Attribute{
 	"Pending":   color.FgHiWhite,
 }
 
+var stepCounter = -1
```

Comment:
should we start at 0?

Reply:
stepsname (as defined in pipeline) start a 0 so it makes the counter increase easier?